services/client/src/styles/_variables.scss

```
@@ -0,0 +1,38 @@
+// ── Colors ──────────────────────────────────────────────────
+
+$color-primary: #0b6394;
+$color-primary-dark: #084b6f;
+$color-success: #2e7d32;
+$color-error: #b91c1c;
+$color-warn: #b45309;
+
+$color-text: #111;
```

Stylelint (17.14.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)